.circleci/config.yml

```diff
@@ -341,9 +341,30 @@ steps-mksnapshot-build: &steps-mksnapshot-build
         name: native mksnapshot (arm/arm64) build
         command: |
           cd src
-          ninja -C out/native_mksnapshot v8:mksnapshot
+          ninja -C out/native_mksnapshot electron:electron_native_mksnapshot_zip
     - store_artifacts:
         path: src/out/native_mksnapshot/mksnapshot
+        destination: native_mksnapshot
+
+steps-mksnapshot-build: &steps-mksnapshot-build
+  steps:
+    - attach_workspace:
+        at: .
+    - *step-depot-tools-add-to-path
+    - *step-setup-env-for-build
+
+    - run:
+        name: mksnapshot GN gen
+        command: |
+          cd src
+          gn gen out/mksnapshot --args='import("//electron/build/args/mksnapshot.gn") cc_wrapper="'"$SCCACHE_PATH"'" $GN_EXTRA_ARGS"
```

Comment:
not 100% sure what changes here occur to differentiate native mksnapshot from mksnapshot

Reply:
Native mksnapshot has to be run as a separate build, but "regular" mksnapshot gets automatically built with Electron (via v8).  steps-mksnapshot-build should just run `ninja -C out/electron electron:electron_mksnapshot_zip`